src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/ReleaseViewModels.cs

```diff
@@ -51,18 +52,31 @@ public class ReleaseViewModel
         [JsonConverter(typeof(StringEnumConverter))]
         public ReleaseType Type { get; set; }
 
-        public Contact Contact { get; set; }
+        public Contact? Contact { get; set; }
 
         [JsonConverter(typeof(StringEnumConverter))]
         public ReleaseApprovalStatus ApprovalStatus { get; set; }
 
         public bool NotifySubscribers { get; set; }
 
-        public string LatestInternalReleaseNote { get; set; }
+        public string? LatestInternalReleaseNote { get; set; }
 
         public bool Amendment { get; set; }
 
         public Guid? PreviousVersionId { get; set; }
+
+        public PermissionsSet? Permissions { get; set; }
+
+        public class PermissionsSet
```

Comment:
Things like this can probably be made a `record` rather than a class, but appreciate this was probably just moved so it's existing code.

Reply:
I've changed all `PermissionsSet` classes across the codebase to `record`s.